src/Game/AccountManagement.java

```diff
@@ -186,13 +412,45 @@ public static void response_display_hook(String text1, String text2) {
         Panel.setControlText(Client.panelRegister, Client.controlRegister, text1 + " " + text2);
       }
 
-      if (Client.login_screen == Client.SCREEN_PASSWORD_RECOVERY) {}
+      if (Client.login_screen == Client.SCREEN_PASSWORD_RECOVERY) {
+    	  Panel.setControlText(Client.panelRecovery, Client.controlRecovery1, text1);
+    	  Panel.setControlText(Client.panelRecovery, Client.controlRecovery2, text2);
+      }
 
       // continue checking if == 2 (login screen) with existing code
     } catch (Exception e) {
       e.printStackTrace();
     }
   }
+  
+  public static boolean pending_render() {
+	  boolean processed = false;
+	  if (Client.showRecoveryQuestions) {
+		  Client.setInterlace(false);
+		  Client.clearScreen();
+		  Panel.drawPanel(Client.panelRecoveryQuestions);
+		  if (customQuestionEntry != -1) {
+	            int n = 150;
+	            Renderer.drawBox(26, n, 460, 60, 0);
+	            Renderer.drawBoxBorder(26, n, 460, 60, 16777215);
+	            n += 22;
+	            Renderer.drawStringCenter("Please enter your question", 256, n, 4, 16777215);
+	            n += 25;
+	            Renderer.drawStringCenter(Client.pm_enteredText + "*", 256, n, 4, 16777215);
+	        }
+		  Renderer.drawSprite(0, Renderer.height_client - 4, Renderer.sprite_media + 22);

```

Comment:
`height_client - 4` may be wrong here. I have just `height_client`

Reply:
yep seems should only be height_client